scanpipe/templates/scanpipe/resource_detail.html

```diff
@@ -35,14 +35,18 @@
     </div>
 
     <section class="mx-5">
-      <div class="message-header">
+      <div class="message-header is-flex">
         <p>
           {{ object.name }}
           | {{ object.size|filesizeformat }}
           | <a class="has-text-white" href="{{ object.get_raw_url }}?as_attachment=1"><i class="fas fa-download"></i></a>
           <br>
           <span style="font-size: 10px;">{{ object.path }}</span>
         </p>
+        <div>
+          <button class="button prev-item is-hidden"><i class="fas fa-arrow-up"></i></button>
```

Comment:
Instead of hiding/showing buttons, it would be better to always display the buttons but use their `disabled` state.

Reply:
sure @tdruez, will try to implement it this way

Reply:
I implemented it over here @tdruez 

https://github.com/nexB/scancode.io/pull/469/commits/cd674fbec747d342fc7c555e8372deff1489d444